### ISSUE TITLE
The CSV reader now always runs the sniffer by default

### ISSUE THREAD

**Issue by szarnyasg** [duckdb/duckdb-web#1417]
See https://github.com/duckdb/duckdb/pull/9250

**Comment by szarnyasg** [comment]
I partially addressed this in #1610.

When the a new stable version is released (that includes the updates `read_csv` function), we should update all pages that are talking about this:
```bash
$ ag --md --ignore-dir=archive/ --files-with-matches read_csv_auto docs/
docs/guides/python/filesystems.md
docs/guides/import/csv_import.md
docs/api/cli.md
docs/api/python/data_ingestion.md
docs/data/overview.md
docs/data/csv/overview.md
docs/data/csv/tips.md
docs/data/csv/auto_detection.md
docs/data/multiple_files/overview.md
docs/data/multiple_files/combining_schemas.md
docs/sql/statements/create_table.md
```
The reason I'm not updating those now is that users often browse the latest (dev) documentation while they are using the stable version. Such a big change would be potentially confusing to them.

**Comment by pdet** [comment]
> I partially addressed this in #1610.
> 
> When the a new stable version is released (that includes the updates `read_csv` function), we should update all pages that are talking about this:
> 
> ```shell
> $ ag --md --ignore-dir=archive/ --files-with-matches read_csv_auto docs/
> docs/guides/python/filesystems.md
> docs/guides/import/csv_import.md
> docs/api/cli.md
> docs/api/python/data_ingestion.md
> docs/data/overview.md
> docs/data/csv/overview.md
> docs/data/csv/tips.md
> docs/data/csv/auto_detection.md
> docs/data/multiple_files/overview.md
> docs/data/multiple_files/combining_schemas.md
> docs/sql/statements/create_table.md
> ```
> 
> The reason I'm not updating those now is that users often browse the latest (dev) documentation while they are using the stable version. Such a big change would be potentially confusing to them.

By default our docs open on the latest release, I think it is fine to add it now, since they need to manually open the `dev` page.
